package-lock.json

```diff
@@ -1,8 +1,11646 @@
 {
   "name": "@permanentorg/node-sdk",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Hm, it appears you are on npm 7, and I am still on npm 6, where running `npm install` rewrites the lockfile and produces a warning:

> npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!

Our build also [uses npm 6](https://github.com/PermanentOrg/node-sdk/pull/26/checks?check_run_id=3000632587#step:3:8), although it seems not to complain when running `npm ci`.